src/infoview.ts

```diff
@@ -188,18 +195,20 @@ export class InfoProvider implements Disposable {
         for (const editor of window.visibleTextEditors) {
             if (editor.document.uri.toString() === uri.toString()) {
                 const pos = new Position(line - 1, column);
-                window.showTextDocument(editor.document);
```

Comment:
What's the rationale for this change here?  Again, please do not comment out, just remove it.

Reply:
Ok, then please remove the line.